CITATION.cff

```diff
@@ -0,0 +1,13 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "ELIXIR"
+title: "RDMkit:The Research Data Management toolkit for Life Sciences"
+version: 2.0.0
+doi: 10.5281/zenodo.5110060
+date-released: 2021-03-01
+url: "https://github.com/elixir-europe/rdmkit"
+preferred-citation:
+  type: generic
+  authors:
+  - family-names: "ELIXIR"
```

Comment:
```suggestion
    - family-names: "ELIXIR"
  year: 2021
  title: "Research Data Management Kit. A deliverable from the EU-funded ELIXIR-CONVERGE project (grant agreement 871075). URL: https://rdmkit.elixir-europe.org"
```

This will render the string almost the same as the one on our About page:
 `ELIXIR. (2021). Research Data Management Kit. A deliverable from the EU-funded ELIXIR-CONVERGE project (grant agreement 871075). URL: https://rdmkit.elixir-europe.org/`

Reply:
This is really optional as it adds weird dots.

Otherwise, we dont need the whole `preffered-citation` block I guess

Reply:
Actually, we should update the one in the about!